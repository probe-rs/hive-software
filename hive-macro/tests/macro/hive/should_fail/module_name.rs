#[hive_macro::hive]
pub mod wrong {}

fn main() {}
