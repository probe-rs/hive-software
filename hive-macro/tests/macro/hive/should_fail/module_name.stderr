error: Top level module for Hive tests must be called tests

         = help: Rename this module to tests
         = note: example:

       #[hive]
       pub mod tests {
           // Testfunctions etc...
       }

 --> tests/macro/hive/should_fail/module_name.rs:2:9
  |
2 | pub mod wrong {}
  |         ^^^^^
